AWSTranscribeStreaming/AWSTranscribeStreamingService.m

```diff
@@ -64,8 +66,8 @@ @interface AWSTranscribeStreaming()
 
 @property (nonatomic, strong) AWSNetworking *networking;
 @property (nonatomic, strong) AWSServiceConfiguration *configuration;
-@property (nonatomic, strong) AWSSRWebSocket *webSocket;
-@property (nonatomic, strong) AWSSRWebSocketDelegateAdaptor *webSocketDelegateAdaptor;
+@property (nonatomic, strong) id<AWSTranscribeStreamingWebSocketProvider> webSocketProvider;
+@property (nonatomic, strong) AWSSRWebSocketDelegateAdaptor *srWebSocketDelegateAdaptor;
```

Comment:
probably should change this to `id<AWSTranscribeStreamingWebSocketProviderDelegate>